app/controllers/hq/users_controller.rb

```diff
@@ -6,4 +6,24 @@ def index
     @users = User.all
   end
 
+  def edit
+    @user = User.find(params[:id])
```

Comment:
Personally I found the Partners code more difficult to follow. When there's a simple statement like on line 10, why complicate it by replacing it with a function call, forcing you to go down to it to follow the code?